config/locales/devise_invitable.en.yml

```diff
@@ -4,6 +4,7 @@ en:
       invited: "You have a pending invitation, accept it to finish creating your account."
     invitations:
       send_instructions: "An invitation email has been sent to %{email}."
+      failed_send: "An error occurred. %{email} could not be invited."
```

Comment:
From a grammar POV, this sounds a little choppy to me. I'm used to see things like

>Error: ${email} could not be invited

or 

>${email} could not be invited (an unknown error occurred)

But it's not up to me @haleymnatz , I just wanted to make a note of it. What does @julianguyen  or @baohouse think?

Reply:
@andy5995 I definitely can change it if that's preferred. I'll just have to get the translations for the changed version as well.

Reply:
You could just rephrase this to `An error occurred, "%{email}" could not be invited.` I would put quotes between "%{email}" in case there is a blank string.